docs/iris/src/userguide/interpolation_and_regridding.rst

```diff
@@ -10,28 +10,45 @@
 Cube regridding and interpolation
 =================================
 
-Iris builds upon interpolation schemes implemented by scipy, and other packages,
-to add powerful cube-aware regrid and interpolation functionality exposed through
-simple cube methods.
+Iris provides powerful cube-aware interpolation and regridding functionality,
+exposed through Iris cube methods. This functionality is provided by building
+upon existing interpolation schemes implemented by SciPy.
+
+In Iris we refer to the avaliable types of interpolation and regridding as
+`schemes`. The following are the interpolation schemes that are currently
+available in Iris:
+ * :class:`iris.analysis.Linear`, and
+ * :class:`Nearest-neighbour<iris.analysis.Nearest>`.
+
+The following are the regridding schemes that are currently available in Iris:
+ * :class:`iris.analysis.Linear`,
+ * :class:`Nearest-neighbour<iris.analysis.Nearest>`, and
+ * :class:`Area Weighted<iris.analysis.AreaWeighted>`.
+
 
 .. _interpolation:
 
 Interpolation
 -------------
 
-Interpolation can be achieved on a cube with the :meth:`~iris.cube.Cube.interpolate`
-method, with the first argument being the points to interpolate, and the second being
-the interpolation scheme to use. The result is a new interpolated cube.
-
-Sample points can be defined as an iterable of ``(coord/coord name, value(s))`` pairs, e.g. ``[('latitude', 51.48), ('longitude', 0)]``.
-The values for coordinates which correspond to date/times may optionally
-be supplied as datetime.datetime or netcdftime.datetime instances,
+Interpolating a cube can be achieved with the :meth:`~iris.cube.Cube.interpolate`
```

Comment:
"can be achieved" --> "is achieved".

...

Feel like I'm doing your job here @bjlittle?? :wink: